lib/rack_middleware/enforce_ssl.rb

```diff
@@ -0,0 +1,21 @@
+class EnforceSsl
```

Comment:
Oops, I asked about this too and we ended up having an offline discussion here. Should have documented it here too.